fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MaterializedViewRewriter.java

```diff
@@ -189,6 +190,7 @@ boolean computeCompatibility(OptExpression queryExpr, OptExpression mvExpr) {
         LogicalOperator queryOp = (LogicalOperator) queryExpr.getOp();
         LogicalOperator mvOp = (LogicalOperator) mvExpr.getOp();
         if (!queryOp.getOpType().equals(mvOp.getOpType())) {
+            logMVPrepare("join type is different {} != {}", queryOp.getOpType(), mvOp.getOpType());
```

Comment:
we can add it into trace info? to better debug? and here it's rewrite phase, so use `logMVRewrite` directly?